config/doctrine.php

```diff
@@ -108,7 +108,6 @@
     |--------------------------------------------------------------------------
     */
     'custom_types'               => [
-        'json' => LaravelDoctrine\ORM\Types\Json::class
```

Comment:
I removed this because this type is already registered by Doctrine, and our own is now deprecated.

https://github.com/doctrine/dbal/blob/2.7/lib/Doctrine/DBAL/Types/Type.php#L83